### DISCUSSION TITLE
Doris与MySQLvarchar长度的问题

### DISCUSSION THREAD

**Discussion by LCehoennardo** [apache/doris#11949] [A - General / Q&A]
从MySQL同步数据到Doris中，varchar类型的字段经常遇到有些中文字符串长度超出doris限制导致同步失败的问题，后来将varchar字段长度扩展为MySQL的三倍，中文问题解决了，但是如果字符串中有emoji，依然会把varchar长度撑爆。这其中的根本原因好像是mysql和doris的varchar长度对于中文和emoji字符的统计方式不一样导致的，想问一下既然doris兼容了mysql协议，后续有没有计划把varchar长度的统计方式也跟mysql对齐呢？

**Comment by fwchitu** [comment]
这个最近也有其他几个用户提到对emoji支持的问题，你说的这个问题，我们看看怎么修复。有进展，会及时在这里同步给你哈。
我们尽量去兼容一下mysql的varchar

**Reply by LCehoennardo** [reply]
好的，多谢哈

**Reply by banyue0618** [reply]
好像两年了，并没有进展？

**Reply by LCehoennardo** [reply]
是啊，官方好像不在乎这个问题
